pegasus/sites.v3/hourofcode.com/i18n/public/el/how-to/districts.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: <%= hoc_s(:title_how_to_districts) %>
+title: '<%= hoc_s(:title_how_to_districts) %>'
```

Comment:
It doesn't matter if this is single or double quote?

Reply:
it sort of does, depending on the content of the translated `title_how_to_districts` string. I'm working on a solution to auto-escape strings that introduce quotes into this field, but for now we have to rely on the https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/test/test_i18n_hoc_routes.rb test

Reply:
So I'm not sure if this is related but I ran into a similar issue when we had to use i18n strings in our JS. I had to use double quotes in JS so I could have ruby concatenated into the string. But then if the translation had a double quote it would break/not know which ones to escape. So I did this hack:

https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/sites.v3/hourofcode.com/views/signup_form.haml#L2

1) Do you still think this hack is necessary? If not, we may be able to remove.

2) Perhaps we can apply this or a more graceful fix more generally across the board for these hoc strings?